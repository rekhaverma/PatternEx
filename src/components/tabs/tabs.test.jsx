import React from 'react';
import Tab from './tabs.component';

describe('<Tab />', () => {
  let wrapper;
  let props;

  const itemList = [
    {
      count: '10',
      id: 'test',
      title: 'Test',
    },
    {
      count: '0',
      id: 'test_tab',
      title: 'Tab',
    },
  ];

  beforeEach(() => {
    props = {
      active: 'test_tab',
      className: 'tab__test',
      customClassName: '',
      fullWidth: false,
      items: itemList,
      style: {},
      slim: false,
      onClick: jest.fn(),
    };
    wrapper = shallow(<Tab {...props} />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

