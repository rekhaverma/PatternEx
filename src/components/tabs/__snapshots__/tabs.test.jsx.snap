// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tab /> should render correctly 1`] = `
<div
  className="tab__test "
  style={Object {}}
>
  <TabCountElement
    active={false}
    className="tab__test__element"
    count="10"
    id="test"
    key="test"
    onClick={[Function]}
    title="Test"
  />
  <TabCountElement
    active={true}
    className="tab__test__element"
    count="0"
    id="test_tab"
    key="test_tab"
    onClick={[Function]}
    title="Tab"
  />
</div>
`;
